ui/src/helpers/httpRequests.js

```diff
@@ -1,7 +1,7 @@
 const axios = require('axios');
-const config = require('./config');
+//const config = require('./config.json');
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.